opl/cluster_read.py

```diff
@@ -373,6 +374,50 @@ def measure(self, ri, name, command, output="text"):
         return name, result
 
 
+class CountLinePlugin(BasePlugin):
+    def check_word_presence(self, word, line):
+        pattern = r"\b{}\b".format(re.escape(word))
```

Comment:
I do not think you need to escape here. Configuration holds the actual regular expression.

Imagine log line like this:

```
error: Something broke
ERROR Another thing broke
```

and config with:

```
log_regexp_error: '(error|ERROR)'
```

(not sure if I need backslashes there or not).

Value of that config option is directly a regular expression.